tests/execution/batching-group-by-optional-key-skip-null.md

```diff
@@ -0,0 +1,64 @@
+# Batching group by
+
+```graphql @config
+schema
+  @server(port: 8000, queryValidation: false)
+  @upstream(baseURL: "http://jsonplaceholder.typicode.com", httpCache: 42, batch: {delay: 1, maxSize: 1000}) {
```

Comment:
```suggestion
  @upstream(baseURL: "http://jsonplaceholder.typicode.com", httpCache: 42, batch: {delay: 1, maxSize: 1000}) {
```

Dont need batching in this test. Please rename the test file and use a simpler example using foo and bar